README.md

```diff
@@ -71,6 +90,22 @@ This mapping can be overridden using the following options:
 * MappingName: Use a data mapping configured in ADX.
 * ColumnsMapping: Use an ingestion-time data mapping.
 
+### Durable Mode
+
+Durable mode can be turned on when we specify the bufferFileName in the LoggerConfiguration. There are few other options available when the durable mode is enabled.
+
+* BufferBaseFileName : Enables the durable mode. When specified, the logs are written to the bufferFileName first and then ingested to ADX.
+
+* BufferFileRollingInterval : The interval at which buffer log files will roll over to a new file. The default is RollingInterval.Hour
+
+* BufferLogShippingInterval : The interval between checking the buffer files.
+
+* BufferFileSizeLimitBytes : The maximum size, in bytes, to which the buffer log file for a specific date will be allowed to grow. By default 100L * 1024 * 1024 will be applied.
+
+* BufferFileLoggingLevelSwitch : A switch allowing the pass-through minimum level to be changed at runtime.
```

Comment:
Add the defaults that we have used in the code, and implications

Reply:
BufferFileLoggingLevelSwitch param doesnt have a default intialization.